src/components/ProjectCard.js

```diff
@@ -1,133 +1,303 @@
 import React from 'react';
-import { makeStyles } from '@material-ui/styles';
-import Avatar from '@material-ui/core/Avatar';
+import { makeStyles, useTheme } from '@material-ui/styles';
 import Card from '@material-ui/core/Card';
 import CardContent from '@material-ui/core/CardContent';
-import Chip from '@material-ui/core/Chip';
 import Grid from '@material-ui/core/Grid';
-import FiberManualRecordRoundedIcon from '@material-ui/icons/FiberManualRecordRounded';
 import githubColorDictionary from './data/gh-colors.json';
+import { Typography } from "@material-ui/core";
+import useMediaQuery from '@material-ui/core/useMediaQuery';
 
 const useStyles = makeStyles((theme) => ({
```

Comment:
The styles section is quite large.  You might want to consider moving it to a separate styles.js file.

Reply:
Created styles.js for ProjectCard component.